example/App.js

```diff
@@ -6,15 +6,48 @@
 
 import { createAppContainer } from 'react-navigation';
 import { createBottomTabNavigator } from 'react-navigation-tabs';
+import { createStackNavigator } from 'react-navigation-stack';
 
 import HomeScreen from "./screens/HomeScreen";
 import SettingsScreen from "./screens/SettingsScreen";
+import MessageListScreen from "./screens/MessageCenterScreen";
+import MessageDetailsScreen from "./screens/MessageScreen";
 
-const TabNavigator = createBottomTabNavigator(
-  {
-    Home: HomeScreen,
-    Settings: SettingsScreen,
-  },
+const MessageCenterStack = createStackNavigator({
+ MessageList: {
+  screen: MessageListScreen,
+  navigationOptions: {
+    headerTitle: 'Messages list',
+  }
+ },
+ MessageDetails: {
+  screen: MessageDetailsScreen,
+  navigationOptions: {
+    headerTitle: 'Message details',
```

Comment:
I'm not sure how to make this change, but the title of this screen should be the title of the message being displayed.

Reply:
Good idea, I fix it 👍